modules/openy_features/openy_media/config/install/image.style.half.yml

```diff
@@ -0,0 +1,14 @@
+langcode: en
```

Comment:
According to naming convention, image style should have prefix of entity `image.style.media_half.yml` and label "Media half".